### DISCUSSION TITLE
Is there way to get lgt8f328p 64pin demo board?

### DISCUSSION THREAD

**Discussion by youxiaojie** [dbuezas/lgt8fx#128] [General]
Is there way to get lgt8f328p 64pin demo board?

**Comment by LaZsolt** [comment]
LGT8F328P maximum number of pins 48
https://detail.1688.com/offer/584944035845.html

**Comment by LaZsolt** [comment]
Another source of a demo board: https://www.keyesrobot.cn/KE0130/

**Comment by LaZsolt** [comment]
And one more source of a board with LGT8F328P LQFP48: https://www.aliexpress.com/item/4001104817179.html

**Comment by LaZsolt** [comment]
And the final one (designed by brother_yan) : https://oshwhub.com/brother_yan/Larduino-QFP48

**Comment by youxiaojie** [comment]
as a chinese I could not find these, need to the help of foreign friend:( ashamed!!

**Reply by LaZsolt** [reply]
Don't worry about it. Different search engines give different result. And more better if you search pictures.
But I could't find any result from Taobao and I could not read it, because no english version.
